src/Evolution/Systems/GeneralizedHarmonic/Equations.hpp

```diff
@@ -127,4 +145,171 @@ struct ComputeNormalDotFluxes {
       const tnsr::II<DataVector, Dim>& inverse_spatial_metric,
       const tnsr::i<DataVector, Dim>& unit_normal) noexcept;
 };
+
+/*!
+ * \ingroup NumericalFluxesGroup
+ * \brief Computes the generalized-harmonic upwind flux
+ *
+ * The upwind flux in general is given by Eq. (6.3) of \cite Teukolsky2015ega :
+ * \f{eqnarray}{
+ * F^* = S \Lambda^{\rm pos} S^{-1} u^{\rm int}
```

Comment:
I don't understand this equation. Where are the spatial indices? The numerical flux has an index. Comparing to Saul's paper this actually is `G`, not `F^*`.

Reply:
This is exactly Eq. (6.3) of the paper I cite, except there was no name given to it. Looks like I should have just called this G here, and then everything's fine?

Reply:
I don't think it should be called G because it's not a flux but a different general boundary correction. It's also not actually a path-conservative boundary correction, which is often denoted by `D` in the literature, but I'd be fine glossing over that and just calling it `D`. The reason I don't like calling it `G` is that in its current form this does not satisfy the Rankine-Hugoniot conditions and so cannot lead to a well-balance scheme (this is just stating in math language what Will's concern is).